lib/craft.ex

```diff
@@ -31,7 +31,17 @@ defmodule Craft do
   @type query() :: any()
   @type log_index() :: non_neg_integer()
 
-  # opts: [:log_module | :data_dir, ...]
+  @doc """
+  Starts a raft group on a list of erlang nodes.
+
+  `nodes` must be an accessible list of erlang nodes, and
+  `machine` must be a module that uses `Craft.Machine` for functionality and implements it's behaviour.
+
+  ### Opts
+    - `:log_module` - Allows caller to provide a custom log module for the nodes in this group.
```

Comment:
i think i just renamed `log_module` to `persistence` at some point. you can delete this